app/src/components/Statistics/Statistics.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from "react";
+import { StatisticsElement } from "./StatisticsElement";
+import { StatisticsData } from "./StatisticsData";
+import {
+  StyledWrapper,
+  StyledHeader,
+  StyledHeading,
+  StyledDescription,
+  StyledStatiscticsElements,
+} from "./Statistics.styles";
+
+export const Statistics = () => {
+  const [Data, setData] = useState(null);
+
+  useEffect(() => {
```

Comment:
State and useEffect is not needed. Just import the StatisticsData and iterate over it. Why did You use state for that and useEffect?

Reply:
Removed.